### PR TITLE
chore/Landing-Page-update

### PR DESCRIPTION
**Pull request by ashutoshamazix**
## Description

- Updated Landing pages

## How Has This Been Tested?

- Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by IAmKio — COMMENTED** [review]
LGTM but just waiting for @RanaBug's approval too

File: src/components/LandingPage/Header/index.jsx
Comment (RanaBug) [inline src/components/LandingPage/Header/index.jsx:44]: If you don't need this in the future, can it be removed please?

File: src/pages/Landing.jsx
Comment (RanaBug) [inline src/pages/Landing.jsx:9]: Same here, if not needed, please remove

**Review by RanaBug — APPROVED** [review]
All good, just some comments that might need to be removed within the code if not needed in the near future

File: src/components/LandingPage/Header/index.jsx
Comment (ashutoshamazix) [reply src/components/LandingPage/Header/index.jsx]: Removed commented-out code.

File: src/pages/Landing.jsx
Comment (ashutoshamazix) [reply src/pages/Landing.jsx]: Removed this.

**Pull request merged**
